src/components/Quote.js

```diff
@@ -0,0 +1,21 @@
+/* eslint-disable max-len */
```

Comment:
- [ ] Please kindly fix your linters errors instead of disabling them. Fix linters help us to write clean and professional codebase 😉